tests/e2e-leg-6/verify-vrpq-conditions/03-setup.yaml

```diff
@@ -11,7 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: kuttl.dev/v1betat1
+apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
-  - command: bash -c "kustomize build setup-vrpq/overlay | kubectl -n $NAMESPACE apply -f - "
+  - script: |
```

Comment:
You don't need to use kubectl here. Just replace the kuttl.dev/v1beta1 YAML manifest with your new CR.